src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -113,17 +115,40 @@ public Future<JsonObject> getItemRecordById(String itemId, boolean skipThrowNorF
   }
 
   /**
-   * Returns list of requests for specified item.
+   * Returns number of requests for specified item.
    *
    * @param itemId id of Item
-   * @return future with list of requests
+   * @return future with number of requests
    */
   public Future<Integer> getNumberOfRequestsByItemId(String itemId, RequestContext requestContext) {
     String query = String.format("(itemId==%s and status=\"*\")", itemId);
     RequestEntry requestEntry = new RequestEntry(INVENTORY_LOOKUP_ENDPOINTS.get(REQUESTS))
       .withQuery(query).withOffset(0).withLimit(0);
     return restClient.getAsJsonObject(requestEntry, requestContext)
-      .map(json -> json.getInteger(TOTAL_RECORDS));
+      .map(json -> json.getInteger(REQUESTS_TOTAL));
+  }
+
+  /**
+   * Returns list of requestIds for specified item.
+   *
+   * @param itemId id of Item
+   * @return future with list of requestIds
+   */
+  public Future<List<String>> getRequestIdsByItemId(String itemId, RequestContext requestContext) {
```

Comment:
let's move these 2 methods connected to Requests into separate service:

- getNumberOfRequestsByItemId

- getRequestIdsByItemId

Reply:
Created it